flake/dev/package-tests.nix

```diff
@@ -1,8 +1,12 @@
+{ self, ... }:
 {
   perSystem =
     { config, ... }:
     {
       # Test that all packages build fine when running `nix flake check`.
       checks = config.packages;
     };
+
+  # Test that all packages build on x86_64-linux when running buildbot
+  flake.ci.buildbot = { inherit (self.packages) x86_64-linux; };
```

Comment:
Same idea here, map over the `whitelist` and include those.

Reply:
Do we really need to build our internal packages on multiple systems in CI?

The only _problematic_ package is `docs` (and anything that depends on it); the others could be built on all systems.

But given they're all internal, I'm not currently motivated to come up with a less "quick and dirty" approach.

Reply:
I think it makes more sense to validate our internal tooling works for all contributors than the test cases that are platform agnostic.

Reply:
Ok. I'll revert this PR back including all internal packages on all systems, then put some effort into restricting only the `docs` packages to one system in a follow up.

Reply:
Done